app/controllers/setup_controller.rb

```diff
@@ -1,11 +1,19 @@
 class SetupController < ApplicationController
 
-  def index
+  before_action do
     @webhook_secret = ENV['WEBHOOK_SECRET']
     if Rails.env.development?
       @webhooks = 'https://$PUBLIC_HOST'+ webhooks_path
     else
       @webhooks = request.base_url + webhooks_path
     end
   end
+
+  def index
+
+  end
+
+  def ci
+
+  end
```

Comment:
No need for empty action methods here, but it doesn't hurt, either.

Reply:
@mathias took out in https://github.com/heroku/webhooks-demo/pull/30/commits/0e20daa39db5513ed540e4eb8997c59f3b234eb9